search2.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Search
+---
+
+<link rel="stylesheet" type="text/css" href="/assets/css/instantsearch.min.css">
+<link rel="stylesheet" type="text/css" href="/assets/css/instantsearch-theme-algolia.min.css">
+
+<script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.10.4"></script>
```

Comment:
oops, missed this. also vendor javascripts?

Reply:
Done